src/pipx/main.py

```diff
@@ -132,7 +132,9 @@ def get_pip_args(parsed_args: Dict[str, str]) -> List[str]:
         pip_args += ["--index-url", parsed_args["index_url"]]
 
     if parsed_args.get("pip_args"):
-        pip_args += shlex.split(parsed_args.get("pip_args", ""))
+        pip_args += shlex.split(
+            parsed_args.get("pip_args", ""), posix="win" not in sys.platform
```

Comment:
wouldn't be more appropriate for this to be `"win32" != sys.platform`

Reply:
@gaborbernat I've tested and using `"win32" != sys.platform` will still output `['D:MODULES_WHEELSDK', 'arg1', 'arg2']`